game.py

```diff
@@ -776,7 +776,7 @@ def do_draw(self, cr):
         # Restrict Cairo to the exposed area
         alloc = self._canvas.get_allocation()
 
-        cr.rectangle(alloc.x, alloc.y, alloc.width, alloc.height)
+        cr.rectangle(alloc.x , alloc.y - style.GRID_CELL_SIZE, alloc.width, alloc.height)
```

Comment:
This the necessary change!! 
A similar change is seen in https://github.com/sugarlabs/sugar/blob/66322ca485485552fbd9c634a58e3f58ea5f5fca/src/jarabe/frame/framewindow.py

And I tested just keeping a change in this line seems to fix the problem! Thanks for helping! :)
Can you confirm if it fixes the issue even in 4:3?

Tested for 16:9, works great! The fix doesn't seem unusual to me , the height of the rectangle is changed. Initially , I guess the the size of the toolbar was not accounted for, therefore the first row was cutoff.

Reply:
Please see the https://github.com/sugarlabs/paths/pull/8#issuecomment-610770716 for screenshots. Feel free to change it / amend it. It was meant to help you